fast_api_als/database/db_helper.py

```diff
@@ -16,6 +16,7 @@
     write a commong function that logs this response code with appropriate context data
 """
 
+logger = logging.getLogger()
```

Comment:
use a Module level logger to get usable logs

Reply:
I am not able to get the difference between logger and direct logging function of module.

Reply:
use logger = logging.getLogger(__name__) in different modules to use a module-level logger

Reply:
__name__

Reply:
Ok. made the changes